### PR TITLE
added sessions 7

### PR DESCRIPTION
**Pull request by Anhal-77**

### CODE REVIEW

File: working_sessions/HOMEWORK-ANHAL.Rmd [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+```{r}
+library(tidyverse)
+library(plotly)
```
Comment (BatoolMM) [inline working_sessions/HOMEWORK-ANHAL.Rmd:14]: why did you add `plotly` if you aren't suing it during the code?

File: working_sessions/HOMEWORK-ANHAL.Rmd [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+```{r}
+
+
+co <- coffee %>%
+  filter(!is.na(variety),
+total_cup_points > 10) %>%
+  mutate(variety = fct_lump(variety, 10),sort = TRUE) 
+
+coffee %>%
```
Comment (BatoolMM) [inline working_sessions/HOMEWORK-ANHAL.Rmd:28]: This is fantastic but you have created above a new variable `co`to store the new dataframe. However, you didn't use it when you created the plot
```suggestion
co %>%
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]
I love the plot, can you try to change the theme and the title of X and y axis

**Comment by Anhal-77** [comment]
> I love the plot, can you try to change the theme and the title of X and y axis

I did it ✅

**Comment by Anhal-77** [comment]
added sessions

**Review by BatoolMM — APPROVED** [review]

**Comment by BatoolMM** [comment]
@all-contributors please add @Anhal-77 for Answering Questions

**Pull request merged**